examples/tests/managed-ng-os-py/__main__.py

```diff
@@ -71,6 +71,27 @@
                                            labels={ "increased-pod-capacity": "true" },
                                            kubelet_extra_args="--max-pods=100")
 
+nodeadm_extra_config = """---
```

Comment:
Ever makes sense to accept dicts here or the YAML literals are more natural for the users?

Reply:
This section can be anything (bash scripts, yaml, etc.), so we'll need to be able to accept all of that.

What I thought of is that we add a function (like we have for IAM policies) for crafting the `NodeConfig` form a typed object.
But that has challenges associated with it. Right now AWS isn't publishing a typed schema for this (the settings are just `object`) and we'll need to come up with a versioning strategy. Tracked here: https://github.com/pulumi/pulumi-eks/issues/1341.

For now I think a plain old string is the most convenient for the users and will cause the least pain in the long run if we end up adding ways to compose the yaml from typed dicts.